### DISCUSSION TITLE
Paid Copilot issue on VS Code

### DISCUSSION THREAD

**Discussion by jahidhasanlinix** [community/community#19447] [Copilot Conversations] [answered]
I have paid the yearly subscription free for github copilot but once I sign it with my github, everytime github copilot is failing i even install few times, close window so on, but its not working? Any suggestion how to fix it, I am using Mac for that. Thanks

**Comment by melvincwng** [comment, marked as answer]
@jahidhasanlinix, try raising a ticket to [GitHub Support](https://support.github.com/contact/account) for them to investigate the issue.

Hope this helps.

**Reply by jahidhasanlinix** [reply]
Thank you, will do then.

**Reply by melvincwng** [reply]
@jahidhasanlinix, if this reply has helped you, please 'Mark as answer' so that the rest are aware this issue has been addressed.

Thanks, and have a great day ahead 😃
